_data/sidebar_tree.yaml

```
@@ -17,6 +17,9 @@ tree:
         title: IMAX B6 Charger
     - url: /kit/brain_board
```

Please can we also update this url so that it matches the actual target url (so that the highlighting of the current page works):
```suggestion
    - url: /kit/brain_board/
```